docs/services/data/configure-cloud-sync.md

```diff
@@ -39,6 +39,12 @@ To change the sync interval, specify an interval in minutes in the interval fiel
 
 {{% /expand%}}
 
+### Pause sync
+
+You can pause Cloud Sync at any time by navigating to the **Services** tab on your robot's **config** tab and disabling **Syncing** for your [Data Management Service](../).
+If you have captured data that you do not want to sync, delete the data on the robot before resuming Cloud Sync.
+To delete the data locally, `ssh` into your robot and delete the data in the directory where you capture data.
```

Comment:
In the above JSON on line 33, could you remove the `capture_dir` field or set it an empty "" value until we change this to handle `~/.viam/capture`?

Perhaps a note here, in the meantime, that if `capture_dir` is unspecified it'll be saved in ~/.viam/capture

Reply:
done!